src/app/(home)/layout.tsx

```diff
@@ -1,26 +1,180 @@
+              a day. Points will convert to $BOSS tokens in June.
+            </Typography>
+          </BlockyCard>
+        </HeroSection>
+        <HeroSectionWithOverflow>
+          <Typography
+            level="h2"
+            className="no-overflow"
+            textColor={"common.white"}
+          >
+            Leaderboard
+          </Typography>
+          <Stack className="overflow">{leaderboardTable}</Stack>
+
+          <Typography
+            className="no-overflow"
+            level="body-sm"
+            sx={{ color: "common.white" }}
+          >
+            Last update on {lastUpdate}. Next update on {nextUpdate}
+          </Typography>
+        </HeroSectionWithOverflow>
+        <HeroSection sx={{ color: "common.white" }}>
+          <Typography level="h2">Become a BOSS</Typography>
+          <Stack
+            sx={{
+              flexDirection: { xs: "column", md: "row" },
+              alignItems: "center",
+              gap: 3,
+            }}
+          >
+            <Stack
+              sx={{
+                height: "100%",
+                flex: 1,
+                alignItems: "center",
+                minHeight: 320,
+                py: 5,
+              }}
+            >
+              <UserShield
+                sx={{ width: 48, height: 48, color: "common.white" }}
+              />
+
+              <Typography level="h3">Step 1</Typography>
+
+              <Typography>
+                Claim your Talent Passport, and increase your Builder Score
+                before the first snapshot, on May 7th at 17:59 UTC.
+              </Typography>
+
+              <Button
+                variant="solid"
+                color="neutral"
+                component={Link}
+                href="https://passport.talentprotocol.com"
+                underline="none"
+                sx={{ mt: "auto" }}
+              >
+                Claim Talent Passport
+              </Button>
+            </Stack>
+
+            <Stack
+              sx={{
+                height: "100%",
+                flex: 1,
+                alignItems: "center",
+                minHeight: 320,
+                py: 5,
+              }}
+            >
+              <UserManLove
+                sx={{ width: 48, height: 48, color: "common.white" }}
+              />
+
+              <Typography level="h3">Step 2</Typography>
+
+              <Typography>
+                Nominate the best builders you know by searching for their web3
+                username or by sending them your personal link.
+              </Typography>
+
+              <Button
+                variant="solid"
+                color="neutral"
+                component={Link}
+                href="/"
+                underline="none"
+                sx={{ mt: "auto" }}
+              >
+                Share Link
+              </Button>
+            </Stack>
+          </Stack>
+        </HeroSection>
       </Stack>
       <Footer />
     </>
```

Comment:
This page has so many abstractions that its no longer worth it to have multiple sections. All the JSX squashed into one file is just 180 lines.